resources/automagic_dashboards/orders.yaml

```diff
@@ -0,0 +1,107 @@
+bindings:
+  fields:
+  - named:
+    - [total, 1.0]
+    - [amount, 0.9]
+    - [income, 1.0]
+    - [price, 0.6]
+    type: Number
+    as: income
+    table: ?orders
+  - named:
+    - [created, 1.0]
+    - [timestamp, 1.0]
+    - [date, 0.8]
+    type: DateTime
+    as: timestamp
+    table: ?orders
+  - named:
+    - [user, 1.0]
+    - [account, 0.9]
+    - [customer, 0.9]
+    - [email, 0.3]
+    type:
+    - [FK, 1.0]
+    - [String, 0.8]
+    as: user
+    table: ?orders
+  - named:
+    - [quantity, 1.0]
+    - [seats, 0.8]
+    - [passengers, 0.8]
+    - [num, 0.2]
+    - [count, 0.5]
+    type: Number
+    as: quantity
+    table: ?orders
+  - named:
+    - [discount, 1.0]
+    type: Number
+    as: discount
+    table: ?orders
+  - named:
+    - [product, 1.0]
+    - [item, 0.8]
+    type:
+    - [FK, 1.0]
+    - [String, 0.8]
+    as: product
+    table: ?orders
+  tables:
+  - named:
+    - [order, 1.0]
+    - [transaction, 1.0]
+    - [sale, 1.0]
+    - [booking, 1.0]
+    as: orders
+dashboards:
+- title: Orders autogenerated
+  as: orders_dashboard
+  description: Autogenerated dashboard
+metrics:
+- title: Average discount
+  as: average_discount
+  query:
+    source_table: "?orders"
+    aggregation: [/, [sum, "?discount"], [sum, "?income"]]
+cards:
+- dashboard: "?orders_dashboard"
+  visualization: line
+  title: Income by month
+  description: Total income by month
+  query:
+    source_table: "?orders"
+    breakout: [datetime-field, "?timestamp", month]
+    aggregation: [sum, "?income"]
+- dashboard: "?orders_dashboard"
+  visualization: line
+  title: Average quantity by month
+  description: Average item quantity by month
+  query:
+    source_table: "?orders"
+    breakout: [datetime-field, "?timestamp", month]
+    aggregation: [avg, "?quantity"]
+- dashboard: "?orders_dashboard"
+  visualization: line
+  title: Average discount by month
+  query:
+    source_table: "?orders"
```

Comment:
I'm not a fan of source table being part of the query language. Field binding is bad enough, having to declare and use tables as variables is making this too much of a stealth DSL.